src/components/interactive/interactive.component.tsx

```diff
@@ -48,7 +49,12 @@ export const Interactive = memo(({ onCoordinateChange, children }: IInteractiveP
   );
 
   return (
-    <div ref={interactiveRef} className="rcp-interactive" onPointerDown={onPointerDown}>
+    <div
+      ref={interactiveRef}
+      className="rcp-interactive"
+      style={{ cursor: disabled ? "not-allowed" : "move" }}
```

Comment:
Let's remove the `style` property and instead use `disabled={disabled}` and update the styles in CSS based on this property.